plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/AbstractTextPanelEditor.java

```diff
@@ -149,6 +149,21 @@ private CLabel createMessageBar(Composite cmpsBase) {
         return messageBar;
     }
 
+    private void hideMessageBar() {
+        GridData gridData = new GridData(SWT.FILL, SWT.FILL, true, false, 1, 1);
+        gridData.heightHint = 0;
+        messageBar.setLayoutData(gridData);
+        messageBar.getParent().layout();
```

Comment:
Use the `GridData.exclude` flag on the existing layout instead of creating a new one every time.

Maybe you don't even need to create that message if the content size doesn't exceed the limit?